packages/core/src/i18n/FA.ts

```diff
@@ -8,7 +8,7 @@ export const FA: ExamTranslations = {
   part: 'Partie {{displayNumber}} :',
   references: {
     heading: 'Sources',
-    source: "D'après",
+    source: "Source :",
```

Comment:
Ton lintti valivalin vois vielä korjata. 

Elikkäs

```suggestion
    source: 'Source :',
```